### ISSUE TITLE
Custom post processing will fail if there is a `'` single quote in subtitle filename. 

### ISSUE THREAD

**Issue by SweetMNM** [morpheus65535/bazarr#1880]
**Describe the bug**
It's broad to say custom post processing will fail because it depends on how you define it.
But from what I've seen it's usually used like so:
```bash
/path/to/important/command '{{subtitles}}'
```
Now you can see how that easily breaks if there is a single quote in the filename,
say for example the movie [One Flew Over the Cuckoo's Nest](https://www.imdb.com/title/tt0073486/).
I can think of some really dodgy workarounds to make it work but it's not really ideal.
The way I see it the feature is practically broken, without extreme workarounds, it will fail.
Unless I am missing some simple way to fix it. Using double quotes is not really a solution since a file might have double quotes in the name as well.

**To Reproduce**
1. Enable some custom post processing script (could be an echo even)
2. Download subtitles for a movie/show with a single quote in the name.
3. Profit???

**Expected behavior**
We should have a way to specify arguments for the custom post processing program,
and make bazarr pass them to the program. Like an editable list of arguments.

OR maybe we should be able to use some special (additional) syntax like:
```bash
/path/to/script ${{subtitles}}
```
which would sanitize the argument (escape all single quotes with a backspace)

**Software (please complete the following information):**
 - Bazarr: 1.0.5-beta.18
 - OS: DietPi on arm64

**Comment by TylerVigario** [comment]
Not familiar with the codebase, but the Movie Title string could be escaped with a backslash. This will cause the ' to not be interpreted by Python.

The string should look like this programmatically

`Movie\'s Title`

**Comment by sambartik** [comment]
It is a bit trickier to support both escaped single and double quotes. What have come to my mind is to include outer quotation marks inside the variables themselves, since the variables are used as parameters to commands anyway. That would allow us to escape inner quotation marks.

What I mean is that `/command {{variable}}` would be replaced for example by: `/command 'This is the content of variable, including quotation marks: (\') and (")'`.

However, this would be a breaking change for users updating to a new version.

**Comment by sambartik** [comment]
It should be fixed in the latest beta. Could you please confirm it fixes this issue?

**Comment by SweetMNM** [comment]
Awesome, I am running the nightly container so as soon as this PR is in an early release I will gladly test and close the issue afterwards. Thanks!

**Comment by morpheus65535** [comment]
@SweetMNM should have been fixed since v1.1.1-beta.4, can you confirm please?

**Comment by SweetMNM** [comment]
@morpheus65535 the issue is fixed 🎉
Thanks a lot!
